src/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_frame_utils.py

```diff
@@ -140,6 +145,7 @@ def map_columns_to_line(self, original_line: str):
 def _utf8_byte_offset_to_character_offset(s: str, offset: int):
     byte_offset = 0
     char_offset = 0
+    offset = offset if offset is not None else 0
```

Comment:
You could just write:

    offset = offset or 0

Reply:
I wondered about that. It works that way in TypeScript, wasn't sure if boolean logic worked that way in Python